Unify Neutronium Plates

ALongStringOfNumbers opened the pull request:
A version of #333 except the commits are not broken by me.

Exaxxion commented:
After rebasing this PR branch to base off the current state of 1.3-PR, loading a world generates a bunch of script errors. See attached log for more information: [crafttweaker.log](https://github.com/Exaxxion/Omnifactory/files/4304042/crafttweaker.log)

ALongStringOfNumbers commented:
So I rebased this pr off the current version of the dev branch as well, and saw the exact same errors, which look like commonVars being initialized, but then erroring the first time it is called in any other script. However, I could not find what was causing the errors. I then created a new instance based of the current state of 1.3-PR and added my changes to those files and then errors did not appear any more, so I don't really know what is going on. I will look into it some more.

ALongStringOfNumbers commented:
The assembler recipe at line ~130 of Endgame.zs is also a redundant recipe, and can be removed from the commits.
